HHS/OPRE-OPS
# Ops 1867 add updated by

## What changed

Added support for `created_by` and `updated_by` to SQLAlchemy.

## Issue

https://github.com/HHS/OPRE-OPS/issues/1867

## How to test

Create an entity in the UI or API and check that the DB `created_by` and `updated_by` are correct.

## Definition of Done Checklist
